frontend/package.json

```diff
@@ -68,16 +68,21 @@
     "@trivago/prettier-plugin-sort-imports": "^4.3.0",
     "eslint": "^8.57.0",
```

Comment:
Needed the bump in package-lock.json to `8.57.1` as the global ignore is otherwise... ignored, which causes linting to happen on build artifacts and that's massively slow.